src/test/java/org/folio/circulation/domain/policy/FixedLoanPolicyCheckOutDueDateCalculationTests.java

```diff
@@ -33,7 +33,7 @@ public void shouldUseOnlyScheduleAvailableWhenLoanDateFits() {
     Loan loan = loanFor(loanDate);
 
     final Result<DateTime> calculationResult = loanPolicy
-      .calculateInitialDueDate(loan);
+      .calculateInitialDueDate(loan, null);
```

Comment:
This null parameter only adds to my increasing sense that there are significant design issues with the calculation of a due date. That is outside the scope of this issue though.